internify/src/components/organisms/styles/AuthNavbar.css

```diff
@@ -0,0 +1,55 @@
+.auth_navbar_container {
+  height: 80px;
+  padding-top: 20px;
+}
+
+.auth_navbar_link {
+  text-decoration: none;
+  color: black;
+}
+
+/* .auth_navbar_item {
+    float: right;
+  } */
+
+.auth_navbar_links {
+  display: flex;
+  flex-direction: row-reverse;
+  margin-top: -30px;
+}
+
+.auth_navbar_item {
+  display: block;
+  text-align: left;
+  /* padding: 10px 10px; */
+  padding-left: 25px;
+  font-size: 20px;
+  font-family: Helvetica;
+}
+
+.auth_navbar_link {
+  text-decoration: none;
+  color: black;
+}
+
+.auth_navbar_logo {
+  left: 100px;
```

Comment:
use material ui to handle styling instead of static px bc you don't know correct screen size